lib/ferrum_pdf/railtie.rb

```diff
@@ -2,13 +2,29 @@ module FerrumPdf
   class Railtie < ::Rails::Railtie
     initializer "ferrum_pdf.assets_helper" do
       ActiveSupport.on_load(:action_view) do
-        include FerrumPdf::AssetsHelper if FerrumPdf.include_assets_helper_module
+        include FerrumPdf::AssetsHelper
       end
     end
 
     initializer "ferrum_pdf.controller" do
       ActiveSupport.on_load(:action_controller) do
-        include FerrumPdf::Controller if FerrumPdf.include_controller_module
+        # render ferrum_pdf: { pdf options }, template: "whatever", disposition: :inline, filename: "example.pdf"
+        ActionController.add_renderer :ferrum_pdf do |pdf_options, options|
+          send_data_options = options.extract!(:disposition, :filename, :status)
+          url = pdf_options.delete(:url)
+          html = render_to_string(**options.with_defaults(formats: [ :html ])) if url.blank?
+          pdf = FerrumPdf.render_pdf(html: html, base_url: request.base_url, url: url, pdf_options: pdf_options)
+          send_data(pdf, **send_data_options.with_defaults(type: :pdf))
```

Comment:
It took me some time to figure this out, but it really is pretty simple. Nice work.